handler/core/src/main/java/io/knotx/fragments/task/factory/TaskOptions.java

```diff
@@ -27,64 +25,24 @@
 @DataObject(generateConverter = true)
 public class TaskOptions {
```

Comment:
Is that class necessary?
It only contains another config inside. There is logic for sugar syntax of `graph` - maybe it could be moved somewhere else?